quixstreams/dataframe/windows/base.py

```diff
@@ -14,6 +14,35 @@ class WindowResult(TypedDict):
 WindowMergeFunc = Callable[[Any], Any]
 
 
+class WindowOnLateCallback(Protocol):
+    def __call__(
+        self,
+        value: Any,
+        timestamp_ms: int,
+        late_by_ms: int,
+        start: int,
+        end: int,
+        store_name: str,
+        topic: str,
+        partition: int,
+        offset: int,
+    ) -> bool: ...
```

Comment:
The user needs to understand why the boolean is expected here and that it controls logging. I think it's rather counterintuitive. That's why in another comment I propose to change it to `-> None` and let users log however they want, and if they want because maybe they don't.